src/it/java/io/deephaven/benchmark/tests/standard/StandardTestRunner.java

```diff
@@ -373,10 +401,11 @@ boolean generateSourceTable(String distribution) {
                 .add("key4", "int", "[0-98]", distribution)
                 .add("key5", "string", "[1-1000000]", distribution)
                 .withRowCount(getGeneratedRowCount())
+                .withColumnGrouping(groups)
```

Comment:
Grouping is how the parquet file is stored for some tests.  The reason I use it is because DataIndex is costly to load ungrouped data, and faster to load grouped data.  So for tests that use DataIndex, I reuse the grouped parquet data to make it load faster, and do not use the grouped data for comparable tests that don't index.  The idea is to not take minutes to load the data scale required to run a meaningful benchmark.

Reply:
After doing more testing on the parquet column grouping (really a sort for now), it appears that not only does pre-sorted data speed up data_index loading, it also speeds up usage, which changes the test significantly.  I've taken it out for now.  So we have benchmarks that run in a fraction of a second.